### PR TITLE
Add wsl contact us

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Check the commit messages

## QA
- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/wsl
- Check that the [highlighted sections of the copy doc](https://docs.google.com/document/d/1M7e2B08GgPA7K6Wx0ThE6s9_xQINM0MSkarfdKUTEYI/edit#) have been applied.
- Click the "Contact us" button and check the interactive contact form matches [the copy doc](https://docs.google.com/document/d/1UxS6S6RTRIiI2rD-yOc8bfbVb2FP1Czd8mGejKrjoDE/edit#heading=h.dxn10eelvkoc)
- Fill in the form and check that works. _Remember the thank you page will load on live so you need to replace the live site domain with local or the demo domain_
- Go to /wsl/contact-us and see that works

## Issue / Card
Fixes https://github.com/canonical-web-and-design/ubuntu.com/issues/8830

### CODE REVIEW

File: templates/wsl/contact-us.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+{% with h1="Contact us about WSL at your company",
+  intro_text="Ubuntu brings you the best-in-class, seamless experience for the Windows Subsystem for Linux. We can help you unleash more capabilities and supply the right level of support you need to have the best experience out-of-the-box.",
+  formid="3755",
+  lpId="",
+  returnURL="https://www.ubuntu.com/wsl/thank-you" %}
+  {% include "shared/_cloud-contact-us-form.html" %}
+{% endwith %}
+
+{% endblock content %}
```
Comment (bethcollins92) [inline templates/wsl/contact-us.html:16]: Not sure if it's important but there is no new line at the end of this file

**Review by bethcollins92 — APPROVED** [review]
LGTM 👍 just one comment about the new line, not sure how important that is.

**Comment by anthonydillon** [comment]
@bethcollins92 thanks, fixed.

**Comment by anasereijo** [comment]
@anthonydillon I thought we decided to make the title `h2` instead of `h3`

**Comment by anthonydillon** [comment]
@anasereijo it is a `h2` but styled as a `h3`. Happy to leave it styled as a `h2` if you think it looks better.

**Comment by anasereijo** [comment]
@anthonydillon I think it's more consistent as it will be the same level as the other titles down the page

**Comment by anthonydillon** [comment]
Thanks @anasereijo can you +1 if your happy with it?

**Comment by anasereijo** [comment]
LGTM 👍

**Pull request merged**